README.md

```diff
@@ -86,7 +95,7 @@ const newUser: User = {
   email: 'john@example.com',
 };
 
-ApiKitClient.post<User>('/users', newUser)
+ApiKitClient.post<User>('/users', newUser, 'json')
```

Comment:
I prefer the third argument to be an object, because later it will give us flexibility to add extra options.

Reply:
Also do we need to have the full known `Content-Type` like: `application/json`, `application/xml` ... ? or default `axios` config request accept your format ?

Reply:
`default axios config`

Reply:
I am what is the default response type `json` or `application/json` ?

Reply:
`application/json`

Reply:
@amrography Done! ✅